tests/Integration/tests/php/interface-with-method/input/index.rst

```diff
@@ -0,0 +1,18 @@
+=====================================
+PHP Interface with implicit namespace
+=====================================
+
+..  php:interface:: TYPO3\CMS\Core\TestInterface
+
+    Lorem Ipsum Dolor!
+
+    ..  php:method:: setDate(int $year, int $month, int $day)
```

Comment:
Is it intended that a trailing comma is rendered after `$day`?

![image](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/737d6635-b579-473a-8da8-e8922673ff22)

Reply:
Good catch! changed it